client/question/question.js

```diff
@@ -251,6 +251,7 @@ tie.constant('SYSTEM_CODE', {
     '        global most_recent_input',
     '        most_recent_input = copy.deepcopy(input)',
     '        output = func(input)',
+    '        print separator',
```

Comment:
Hm -- I think it would be better for the _caller_ of this function to print the separator instead. Given its name, this function should probably just return the output without side-effects.

Reply:
Done. Moved to PythonCodePreprocessor